src/sass/static/pages.scss

```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
         margin: 0 auto;
     }
 }
+
+.careers-content {
+    .career {
+        margin-top: 0;
+
+        .top-banner {
+            background: url('../images/pages/careers/careers-1_bg@3x.jpg') no-repeat;
+            background-size: cover;
+            min-height: 300px;
+            position: relative;
+            background-position: center center;
+
+            .banner-caption {
+                position: absolute;
+                z-index: 1;
+                width: 100%;
+                bottom: -100px;
+
+                .caption-text {
+                    margin: 0 auto;
+                    max-width: 960px;
+                    background-color: $COLOR_WHITE;
+                    text-align: center;
+                    padding: 40px 0px 20px;
+
+                    p {
+                        margin: 1em auto;
+                        max-width: 80%;
+                    }
+                }
+            }
+            @media (max-width: 768px) {
+                background-position: 75% -30px;
+            }
+        }
+        .center-banner {
+            background: url('../images/pages/careers/careers-2_bg@3x.jpg') no-repeat;
+            background-size: cover;
+            min-height: 300px;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            background-position: center center;
+
+            .caption-text {
+                margin: 0 auto;
+                max-width: 960px;
+                text-align: center;
+
+                p {
+                    color: $COLOR_WHITE;
+                    margin: 1em auto;
+                    max-width: 80%;
+                    font-weight: 300;
+                }
+            }
+        }
+        .margin-top-100 {
+            margin-top: 100px;
+        }
+        .no-margin-top {
+            margin: 0 0 1em;
```

Comment:
from the class name it sounds like the style should be `margin-top: 0;`, can we change the style or update the class name?